shanoir-ng-front/src/app/examinations/examination/examination.component.html

```diff
@@ -180,14 +180,11 @@ <h2 class="header command-zone"i18n="Create examination|Title@@examinationDetail
 					(delete) = "delete()"
 					(cancel)="goToView()"
 					(back)="goBack()">
-				<button *ngIf="mode == 'view' && hasDownloadRight && !noDatasets && hasEEG" [disabled]="progressBar.progress != 0 || noDatasets" type="button" class="right-icon" i18n="Buttons|Download button label@@downloadButton" (click)="download('eeg')">Download EEG<i class="fas fa-download"></i></button>
-                <button *ngIf="mode=='view' && hasDownloadRight && hasBids" [disabled]="progressBar.progress != 0 || noDatasets" type="button" class="right-icon" i18n="Buttons|Download button label@@downloadButton" (click)="download('BIDS')">Download BIDS dataset file<i class="fas fa-download"></i></button>
+				<button *ngIf="mode == 'view' && hasDownloadRight && !noDatasets && hasEEG" [disabled]="noDatasets" type="button" class="right-icon" i18n="Buttons|Download button label@@downloadButton" (click)="download('eeg')">Download EEG<i class="fas fa-download"></i></button>
```

Comment:
Could we make a single button ? This message box is not really usefull
![image](https://github.com/fli-iam/shanoir-ng/assets/7604176/11ec6ffa-9ab0-49da-a61e-136a2ae8bc92)

Reply:
And the select box option is either nifti or dicom? (if EEG, download it anyways for example)

Reply:
* The window will hopefully provide more options in future. 

* Sorry I don't understand your question.

Reply:
Mmm, the idea here is that the popup is quite useless here for the moment.
So the idea would be to remove the two buttons "download as nifti" and "download as dicom", to make a single button "download" that open the new popup window with the dicom/nifti choice